cmd/client/lib/cli.go

```diff
@@ -122,11 +122,13 @@ func Create(c *cli.Context, withInstrumentation bool, opts ...client.Option) (cl
 		opts = append(opts, client.WithChainInfo(info))
 	}
 
-	gc, err := buildGrpcClient(c, &info)
+	gc, info, err := buildGrpcClient(c, info)
 	if err != nil {
 		return nil, err
 	}
-	clients = append(clients, gc...)
+	if len(gc) > 0 {
```

Comment:
should `len(gc) == 0` be an error? I can't see it happening in the lower scope function anyway unless there's an error - in that case would `gc...` be a no-op?

Reply:
If we make `len(gc) == 0` then we can't start the relay without a gRPC connection. The check is there because we now might return a `nil` instead of an empty `[]grpc.Client` slice from the above function.